cmd/format-coverage.go

```diff
@@ -52,6 +53,7 @@ var formatterMap = map[string]formatters.Formatter{
 	"lcov-json":   &lcovjson.Formatter{},
 	"simplecov":   &simplecov.Formatter{},
 	"xccov":       &xccov.Formatter{},
+	"dotcover":       &dotcover.Formatter{},
```

Comment:
This got formatted a little weird. Do we run `gofmt` against this repo?

Reply:
Good catch, `codeclimate` is now set up on this repo 😉